test/integration/install_test.go

```diff
@@ -1248,6 +1248,19 @@ func TestRestarts(t *testing.T) {
 	}
 }
 
+func TestInstallCleanUp(t *testing.T) {
+	ctx := context.Background()
```

Comment:
Hmm, I'm not extremely clear on what we are doing here. At first, I thought we are testing the helper logic of deleting a namespace, but looking at it a bit more closely it looks as if we are cleaning up the smoke test namespaces?

Reply:
Yep, This cleanup logic is currently in the `tracing_test` which I think is confusing? (I think we added that there because thats where the test was failing because the cluster has limited cpu/memory preventing components from the tracing test spinning up). I moved here so that its cleaned after the test cases are done in `install_test`

Reply:
I wonder if we can change the name slightly 🤔 at first to me it seemed like we're just testing the utility. Do you think it would be possible to run each test case within its own namespace instead of having the clean-up logic separate? Something similar to what we're doing in the opaque test, not sure if it's going to be super easy though because the injection cases is more of a table-style of testing but curious to see what your thoughts are.

Reply:
>  not sure if it's going to be super easy though because the injection cases is more of a table-style of testing but curious to see what your thoughts are.

Yeah. I'm considering it would be a much more bigger change, and also out-of-scope for this PR specifically? I'm +1 to the idea of composing this in similar manner like we do with others. I might take a stab at this change after this.

Reply:
Meanwhile, I updated the method to be called `TestCleanUp` and added a comment fro the same. It is indeed a bit confusing but `Test` prefix is necessary for it to be ran!